src/server/game/Entities/Object/Object.h

```diff
@@ -540,6 +540,14 @@ struct Position
 
     float GetAngle(const Position* pos) const;
     [[nodiscard]] float GetAngle(float x, float y) const;
+    float GetAbsoluteAngle(float x, float y) const
+    {
+        float dx = x - m_positionX;
+        float dy = y - m_positionY;
+        return NormalizeOrientation(std::atan2(dy, dx));
+    }
+    float GetAbsoluteAngle(Position const& pos) const { return GetAbsoluteAngle(pos.m_positionX, pos.m_positionY); }
+    float GetAbsoluteAngle(Position const* pos) const { return GetAbsoluteAngle(*pos); }
```

Comment:
please always add `[[nodiscard]]` to all methods that return something without causing any side-effects